.claude/skills/icon.md

```
@@ -0,0 +1,31 @@
+
+アイコンライブラリの使用は禁止。必ず以下の手順でGoogle FontsからAndroid Vector DrawableのXMLを取得する。
+
+SVGはAndroidで直接使用できないため、XML形式で取得すること。
+
+## 手順
+
+1. https://fonts.google.com/icons?icon.query=<検索キーワード> でアイコンを検索する
+2. 使用するアイコン名（例: `home`、`add_a_photo`）を確認する
+3. 以下のURLパターンでGitHubから直接XMLをダウンロードする
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)